app.py

```diff
@@ -51,6 +51,13 @@ def create_app() -> FlaskApp:
         resolver=MethodViewResolver("api"),
     )
 
+    connexion_app.add_api(
+        get_bundled_specs("/account_store/openapi/api.yml"),
+        validate_responses=False,
```

Comment:
Note this defaults to `None` which will set it to false, this is in the spirit of lifting and shifting.

Reply:
i can't believe you've done this.gif